Update deployment-overview.md (#2483)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/deployment-overview.md

## Prefect

> If your organization is using [Prefect]([url](https://www.prefect.io/)), the way you will run your jobs depends on the version you are on, and whether you are orchestrating dbt Cloud or core jobs. A variety of options are described below.

Reviewer: sugg clarifying whether it's dbt version.

```suggestion
If your organization is using [Prefect](https://www.prefect.io/), the way you will run your jobs depends on the dbt version you're on, and whether you're orchestrating dbt Cloud or dbt Core jobs. 

Review a variety of options described below.
```

> <Lightbox src="https://user-images.githubusercontent.com/12350579/186269036-01f1090f-1887-433d-8331-ee7c8b315dd5.png" title="Prefect DAG using a dbt Cloud job run flow"/> 

Reviewer: updating image url

```suggestion
<Lightbox src="/img/docs/running-a-dbt-project/prefect_dag_dbt_cloud.jpg" title="Prefect DAG using a dbt Cloud job run flow"/> 
```

### On Prefect 2

#### dbt Cloud

> Use the [trigger_dbt_cloud_job_run_and_wait_for_completion](https://prefecthq.github.io/prefect-dbt/cloud/jobs/#prefect_dbt.cloud.jobs.trigger_dbt_cloud_job_run_and_wait_for_completion) flow. As jobs are executing, you can poll dbt to see whether or not the job completes without failures, through the [Prefect UI](https://docs.prefect.io/ui/overview/).

Reviewer: ```suggestion
Use the [trigger_dbt_cloud_job_run_and_wait_for_completion](https://prefecthq.github.io/prefect-dbt/cloud/jobs/#prefect_dbt.cloud.jobs.trigger_dbt_cloud_job_run_and_wait_for_completion) flow. As jobs are executing, you can poll dbt to see whether or not the job completes without failures, through the [Prefect user interface (UI)](https://docs.prefect.io/ui/overview/).

```

> Use the [trigger_dbt_cloud_job_run_and_wait_for_completion]([url](https://prefecthq.github.io/prefect-dbt/cloud/jobs/#prefect_dbt.cloud.jobs.trigger_dbt_cloud_job_run_and_wait_for_completion)) flow. As jobs are executing, you can poll dbt to see whether or not the job completes without failures, through the [prefect UI]([url](https://docs.prefect.io/ui/overview/)).

Reviewer: hey @belasobral93 fixing the url as the deploy link was directing this to a 'page not found'. below is the URL guidance:

❌ [prefect UI]([url](https://docs.prefect.io/ui/overview/)).
✅ [prefect UI](https://docs.prefect.io/ui/overview/).

```suggestion
Use the [trigger_dbt_cloud_job_run_and_wait_for_completion](https://prefecthq.github.io/prefect-dbt/cloud/jobs/#prefect_dbt.cloud.jobs.trigger_dbt_cloud_job_run_and_wait_for_completion) flow. As jobs are executing, you can poll dbt to see whether or not the job completes without failures, through the [Prefect UI](https://docs.prefect.io/ui/overview/).
```

> <Lightbox src="https://user-images.githubusercontent.com/35778239/202576940-79adcc08-6412-4749-83ad-f0b66ba4cddc.png" title="dbt Cloud job triggered by Prefect"/> 

Reviewer: updating image url

 ```suggestion
<Lightbox src="/img/docs/running-a-dbt-project/dbt_cloud_job_prefect.jpg" title="dbt Cloud job triggered by Prefect"/> 
```

#### dbt Core

> Use the [trigger_dbt_cli_command]([url](https://prefecthq.github.io/prefect-dbt/cli/commands/#prefect_dbt.cli.commands.trigger_dbt_cli_command)) task.

Reviewer: ```suggestion
Use the [trigger_dbt_cli_command](https://prefecthq.github.io/prefect-dbt/cli/commands/#prefect_dbt.cli.commands.trigger_dbt_cli_command) task.
```

> For details on both of these methods, see [prefect-dbt docs]([url](https://prefecthq.github.io/prefect-dbt/)).

Reviewer: ```suggestion
For details on both of these methods, see [prefect-dbt docs](https://prefecthq.github.io/prefect-dbt/).
```

### On Prefect 1

#### dbt Cloud

> Trigger dbt Cloud jobs with the [DbtCloudRunJob]([url](https://docs.prefect.io/api/latest/tasks/dbt.html#dbtcloudrunjob)) task. Running this task will generate a markdown artifact viewable in the Prefect UI. The artifact will contain links to the dbt artifacts generate as a result of the job run.

Reviewer: i think this is meant to be generated?

```suggestion
Trigger dbt Cloud jobs with the [DbtCloudRunJob](https://docs.prefect.io/api/latest/tasks/dbt.html#dbtcloudrunjob) task. Running this task will generate a markdown artifact viewable in the Prefect UI. The artifact will contain links to the dbt artifacts generated as a result of the job run.
```

#### dbt Core

> Use the [DbtShellTask]([url](https://docs.prefect.io/api/latest/tasks/dbt.html#dbtshelltask)) to schedule, execute and monitor your dbt runs. Use the supported [ShellTask]([url](https://docs.prefect.io/api/latest/tasks/shell.html#shelltask)) to execute dbt commands through the shell.

Reviewer: ```suggestion
Use the [DbtShellTask](https://docs.prefect.io/api/latest/tasks/dbt.html#dbtshelltask) to schedule, execute, and monitor your dbt runs. Use the supported [ShellTask](https://docs.prefect.io/api/latest/tasks/shell.html#shelltask) to execute dbt commands through the shell.
```
